lib/iris/fileformats/pp_rules.py

```diff
@@ -968,30 +980,78 @@ def _all_other_rules(f):
 
     # Cross-sectional time values (--> vector coordinates)
     if (len(f.lbcode) == 5 and f.lbcode[-1] == 1 and f.lbcode.iy == 23):
+        epoch_hours_unit = f.time_unit('hours')
         dim_coords_and_dims.append(
             (DimCoord(
                 f.y,
                 standard_name='time',
-                units=Unit('days since 0000-01-01 00:00:00',
+                units=Unit(epoch_hours_unit,
```

Comment:
It is intentional; it brings the time coordinate units in line with the additions I've made for `lbcode=3xxxx`. If setting the epoch like this is appropriate for `lbcode=3xxxx` then I believe it is also appropriate for `lbcode=1xxxx`, which is ostensibly identical to `lbcode=3xxxx`, certainly where it comes to the unit of the time coordinate.